.gitignore

```diff
@@ -24,3 +24,4 @@ clover.xml
 /.project
 /.buildpath
 /.settings
+src/.main/Main.php
```

Comment:
This should probably not be checked in to the repo and instead remain for your local .gitignore only.